src/5e-SRD-Backgrounds.json

```diff
@@ -0,0 +1,112 @@
+                    "name": "Dwarvish",
+                    "url": "/api/languages/dwarvish"
+                },
+                {
+                    "index": "elvish",
+                    "name": "Elvish",
+                    "url": "/api/languages/elvish"
+                },
+                {
+                    "index": "giant",
+                    "name": "Giant",
+                    "url": "/api/languages/giant"
+                },
+                {
+                    "index": "gnomish",
+                    "name": "Gnomish",
+                    "url": "/api/languages/gnomish"
+                },
+                {
+                    "index": "goblin",
+                    "name": "Goblin",
+                    "url": "/api/languages/goblin"
+                },
+                {
+                    "index": "halfling",
+                    "name": "Halfling",
+                    "url": "/api/languages/halfling"
+                },
+                {
+                    "index": "orc",
+                    "name": "Orc",
+                    "url": "/api/languages/orc"
+                },
+                {
+                    "index": "abyssal",
+                    "name": "Abyssal",
+                    "url": "/api/languages/abyssal"
+                },
+                {
+                    "index": "celestial",
+                    "name": "Celestial",
+                    "url": "/api/languages/celestial"
+                },
+                {
+                    "index": "draconic",
+                    "name": "Draconic",
+                    "url": "/api/languages/draconic"
+                },
+                {
+                    "index": "deep-speech",
+                    "name": "Deep Speech",
+                    "url": "/api/languages/deep-speech"
+                },
+                {
+                    "index": "infernal",
+                    "name": "Infernal",
+                    "url": "/api/languages/infernal"
+                },
+                {
+                    "index": "primordial",
+                    "name": "Primordial",
+                    "url": "/api/languages/primordial"
+                },
+                {
+                    "index": "sylvan",
+                    "name": "Sylvan",
+                    "url": "/api/languages/sylvan"
+                },
+                {
+                    "index": "undercommon",
+                    "name": "Undercommon",
+                    "url": "/api/languages/undercommon"
+                }
+            ]
+        },
+        "starting_equipment": "/api/starting-equipment/acolyte",
+        "feature": {
+            "index": "shelter-of-the-faithful",
+            "name": "Shelter of the Faithful",
+            "url": "/api/features/shelter-of-the-faithful"
+        },
+        "suggested_characteristics": "api/suggested-characteristics/acolyte",
+        "url": "api/backgrounds/acolyte"
+    }
+]
```

Comment:
Nit pick: Add new line at the end of the files that don't have it. If you use an IDE or code editor, there should be a setting to do this.